lighthouse-core/report/templates/report-templates.js

```diff
@@ -0,0 +1,283 @@
+this["report"] = this["report"] || {};
```

Comment:
That is debatable. If we add in gitignore,  we need to make sure that it will be generated on the fly for people who just clones the repo and runs node-cli command.